src/lock-fields-manager.ts

```diff
@@ -458,4 +480,22 @@ export class LockFieldsManager<U extends BaseMongoObject> {
 			return excludedField === path;
 		});
 	}
+
+	private getArrayReferencesForPath(path: string): string[] | undefined {
+		const arrayWithReferencesForPath = this.conf.arrayWithReferences?.[path];
+		if (arrayWithReferencesForPath) return _.castArray(arrayWithReferencesForPath);
```

Comment:
this castArray could be done in the constructor to save time in runtime

Reply:
@BenjD90 done!